src/main/java/com/matt/forgehax/mods/infooverlay/Watermark.java

```diff
@@ -0,0 +1,115 @@
+package com.matt.forgehax.mods.infooverlay;
+
+import static com.matt.forgehax.Helper.printError;
+
+import com.matt.forgehax.util.command.Setting;
+import com.matt.forgehax.util.mod.Category;
+import com.matt.forgehax.util.mod.HudMod;
+import com.matt.forgehax.util.mod.loader.RegisterMod;
+import com.matt.forgehax.util.draw.SurfaceHelper;
+import com.matt.forgehax.util.math.AlignHelper.Align;
+import com.matt.forgehax.util.color.Colors;
+import com.matt.forgehax.util.color.Color;
+import net.minecraftforge.client.event.RenderGameOverlayEvent;
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
+import com.matt.forgehax.asm.events.PacketEvent;
+import net.minecraft.network.play.server.SPacketTimeUpdate;
+
+@RegisterMod
+public class Watermark extends HudMod {
+
+  public Watermark() {
+    super(Category.GUI, "Watermark", true, "Display a watermark on your screen");
+  }
+
+  private int color = 0;
+
+  private final Setting<Integer> alpha =
+      getCommandStub()
+          .builders()
+          .<Integer>newSettingBuilder()
+          .name("alpha")
+          .description("Transparency, 0-255")
+          .min(0)
+          .max(255)
+          .defaultTo(255)
+          .build();
+  private final Setting<Integer> red =
+      getCommandStub()
+          .builders()
+          .<Integer>newSettingBuilder()
+          .name("red")
+          .description("Red amount, 0-255")
+          .min(0)
+          .max(255)
+          .defaultTo(0)
+          .build();
+  private final Setting<Integer> green =
+      getCommandStub()
+          .builders()
+          .<Integer>newSettingBuilder()
+          .name("green")
+          .description("Green amount, 0-255")
+          .min(0)
+          .max(255)
+          .defaultTo(0)
+          .build();
+  private final Setting<Integer> blue =
+      getCommandStub()
+          .builders()
+          .<Integer>newSettingBuilder()
+          .name("blue")
+          .description("Blue amount, 0-255")
+          .min(0)
+          .max(255)
+          .defaultTo(0)
+          .build();
+
+  private final Setting<String> text =
+    getCommandStub()
+      .builders()
+      .<String>newSettingBuilder()
+      .name("text")
+      .description("Watermark text")
+      .defaultTo("ForgeHax 2.10.0")
+      .build();
+
+  private final Setting<Boolean> rainbow =
+    getCommandStub()
+      .builders()
+      .<Boolean>newSettingBuilder()
+      .name("rainbow")
+      .description("Change color every 4 ticks")
```

Comment:
descr inaccurate, its based on `SPacketTimeUpdate` in the code below?

Reply:
Whops, don't know what I was thinking! TimeUpdate is every 20 ticks

Reply:
Watermark mod is now called ForgeHaxService, holds mainly general client settings